src/plugins/intel_gpu/src/kernel_selector/kernels/fully_connected/fully_connected_kernel_bf_tiled.cpp

```diff
@@ -233,8 +233,17 @@ bool TuneParamsSelector::VerifyTuneParams(const fully_connected_params& params,
         return false;
 
     auto batch_size = params.is_shape_agnostic ? Align(output_b, tparams.tile_b) : output_b;
-    if (batch_size % (tparams.tile_b * tparams.dispatch_bsv) != 0)
-        return false;
+    // If batch size is prime number, still can apply tile execution to avoid poor performance.
```

Comment:
Could you let me know how the baseline assumption of "prim number bsize" is good for the batch tile? 
Can we make sure that the accuracy is okay?

Reply:
"prim number bsize" is observed in some ViT models, which causes FC `tuneParams`
 fallback to `tune_params(1, 1, 1, 1, 1, 1, EXE_MODE_DEFAULT)`, and lead to very poor performance due to data-read utilization is very low. 
We notice the FC kernel implementation support unaligned shape size(2/4/8/16...), so for such prime number shape, we also can choose better `tuneParams` to improve performance. Of course, we didn't want to effect original `tuneParams` result, so only check and apply it for prime number shape.

Yes, there is a test case to verify the accuracy, and we didn't obverse any accuracy issue.